pymc3/tests/test_distributions.py

```diff
@@ -415,7 +425,7 @@ def test_wald(self, value, mu, lam, phi, alpha, logp):
         with Model() as model:
             Wald('wald', mu=mu, lam=lam, phi=phi, alpha=alpha, transform=None)
         pt = {'wald': value}
-        assert_almost_equal(model.fastlogp(pt), logp, decimal=6, err_msg=str(pt))
+        assert_almost_equal(model.fastlogp(pt), logp, decimal=select_decimal(float64=6, float32=1), err_msg=str(pt))
```

Comment:
IMHO yes, a relative tolerance would be more appropriate in several cases like this.

IMHO we might as well make a separate issue because there's likely to be 10 more PRs worth of precision fixes to get GPU tests passing.